src/utils/chem-utils.ts

```diff
@@ -85,9 +183,34 @@ export const chemistryTests: ChemistryTest[] = [
              {value: 20, rating: ChemTestRatingType.poor, color: "#dc8c74"}, {value: 40, rating: ChemTestRatingType.poor, color: "#d24116"}]
   },
   { type: ChemTestType.turbidity, label: t("CHEM.TURBIDITY.TEST"), units: t("CHEM.TURBIDITY.UNIT"),
-    steps: [{type: StepType.animation, label: t("CHEM.COLLECT.SAMPLE")}, {type: StepType.resultSlider, label: t("CHEM.MATCH.VALUE")}],
-    results: [{value: 0, rating: ChemTestRatingType.excellent, icon: Turbidity0}, {value: 40, rating: ChemTestRatingType.good, icon: Turbidity40},
-             {value: 100, rating: ChemTestRatingType.fair, icon: Turbidity100}]
+    InitialImage: Turbidity0,
+    steps: [
+      {type: StepType.animation, label: t("CHEM.COLLECT.SAMPLE"), Image: "1f",
+        frames: [
+          {label: "0", image: Turbidity0, duration: 0.25},
+          {label: "1a", image: Turbidity1A, duration: 1},
+          {label: "1b", image: Turbidity1B, duration: 2},
+          {label: "1c", image: Turbidity1C, duration: 0},
+          {label: "1d", image: Turbidity1D, duration: 2},
+          {label: "1d-", image: "none", duration: 0},
+          {label: "1e", image: "byValue", duration: 1},
+          {label: "1f", image: "byValue", duration: -1},
```

Comment:
Did we check to see if there are cases where we need to store a fade time as well?

Reply:
I didn't look through all the tests, but there were no other fade times in the tests I looked at.

Reply:
I'm mostly thinking of the case where we mix the sample.  I don't see it in the spec, but it clearly uses different fade timing in the proto:
![mix](https://user-images.githubusercontent.com/5126913/108604607-d22c3080-7363-11eb-9232-5574957a7a73.gif)

Reply:
I just looked and the spec has them all at 0.5 sec so I figure we start there and if we have to make it more elaborate we can do so.

Reply:
Sounds good.